cliquet/permission/postgresql/__init__.py

```diff
@@ -163,6 +163,30 @@ def object_permission_authorized_principals(self, object_id, permission,
             results = cursor.fetchall()
         return set([r['principal'] for r in results])
 
+    def principals_accessible_objects(self, principals, permission,
+                                      object_id_match=None):
+        if object_id_match is None:
+            object_id_condition = 'true'
+        else:
+            object_id_match = object_id_match.replace('*', '%%')
+            object_id_condition = "object_id LIKE '%s'" % object_id_match
```

Comment:
Oh actually the %% are escaped if we put them in a placeholder. Also the risk for SQL injection here is inexistant because this parameter is not coming from a user input.